rotkehlchen/chain/ethereum/interfaces/ammswap/types.py

```diff
@@ -56,7 +56,24 @@ def serialize(self) -> Dict[str, Any]:
         }
 
 
+@dataclass(init=True, repr=True)
+class NFTLiquidityPool(LiquidityPool):
+    nft_id: int
+    price_range: Tuple[FVal, FVal]
+
+    def serialize(self) -> Dict[str, Any]:
+        return {
+            'address': self.address,
+            'nft_id': self.nft_id,
+            'price_range': f'{self.price_range[0]}<>{self.price_range[1]}',
```

Comment:
Why format the price ranges like this and not just return the tuple? Probably any service consuming this information would need to split the string

Reply:
Okay.